jsonschema/exceptions.py

```diff
@@ -43,6 +44,7 @@ def __init__(
             schema,
             schema_path,
             parent,
+            json_path,
```

Comment:
This seems like it's a computed attribute (via `property`) -- it doesn't seem like it should be either in `__init__` or `_contents`, no?

Reply:
I've now removed them from both places